src/Jenkins/Build.php

```diff
@@ -65,13 +65,20 @@ public function getInputParameters()
     {
         $parameters = array();
 
-        if (!property_exists($this->build->actions[0], 'parameters')) {
-            return $parameters;
-        }
-
-        foreach ($this->build->actions[0]->parameters as $parameter) {
-            $parameters[$parameter->name] = $parameter->value;
-        }
+		// Parameters may not always be the first definition. Search for 'hudson.model.ParametersAction' to find
+		// real parameters.
+		foreach ($this->build->actions as $action) {
+			if (isset($action->_class) && $action->_class === 'hudson.model.ParametersAction') {
```

Comment:
Why do you need this condition ? With this one, I can't retrieve any parameters from my build (on jenkins 1.656)